### PR TITLE
Fabric Operator Labs Proposal

### PR DESCRIPTION
**Pull request by jkneubuh**
This PR is a lab proposal for **fabric-operator**, a cloud-native operator for building, administering, and maintaining Hyperledger Fabric networks on Kubernetes.

Signed-off-by: Josh Kneubuhl <jkneubuh@us.ibm.com>

### CODE REVIEW

File: labs/fabric-operator.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- A web-based User interface, provided by the companion [Fabric Operations Console](https://github.com/hyperledger-labs/fabric-operations-console)
+- Declarative, [Ansible automation playbooks](https://github.com/IBM-Blockchain/ansible-collection) 
+
+
+**fabric-operator** is an open-source, production-grade, community contribution from IBM.  The work spans several
+(hundreds of?) thousands of hours of hard work, ingenuity, passion, and dedication from the team.  Special thanks
+are in order for key contributors:
+
+
+# Initial Committers
```
Comment (tkuhrt) [inline labs/fabric-operator.md:60]: In this section, please include the GitHub IDs for the committers. This section is what we will use to add people to the correct team within GitHub.

File: labs/fabric-operator.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- [Ratnakar Asara](https://github.com/asararatnakar)
+- Jamie Min
+- [Gari Singh](https://www.linkedin.com/in/garisingh/)
+
+
+# Sponsors
+
+- [David Boswell](https://github.com/todo) - Hyperledger XYZ
+- [Hart Montgomery](https://github.com/todo) - Hyperledger XYZ 
+- [Ry Jones](https://github.com/todo) - Hyperledger XYZ
```
Comment (tkuhrt) [inline labs/fabric-operator.md:75]:
```suggestion
- [David Boswell](https://github.com/todo) - Hyperledger Foundation
- [Hart Montgomery](https://github.com/hartm) - Hyperledger Foundation 
- [Ry Jones](https://github.com/ryjones) - Hyperledger Foundation
```

File: labs/fabric-operator.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+title: Fabric Operator
+parent: Labs
+---
+# Lab Name
+Hyperledger Fabric Operator
+
+# Short Description
+**fabric-operator** is an open-source, cloud-native [Kubernetes operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+for administering Hyperledger Fabric networks.  The operator follows the [CNCF Operator Pattern](https://github.com/cncf/tag-app-delivery/blob/main/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md),
+delegating the minutia of performing repetitive, detailed configuration tasks to automated activities performed under
```
Comment (vipinsun) [inline labs/fabric-operator.md:12]: minutiae

File: labs/fabric-operator.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+# Sponsors
+
+- [David Boswell](https://github.com/todo) - Hyperledger XYZ
+- [Hart Montgomery](https://github.com/todo) - Hyperledger XYZ 
+- [Ry Jones](https://github.com/todo) - Hyperledger XYZ
+- [Dave Enyeart](https://github.com/denyeart)  - Hyperledger TSC Member
+- [Arnaud LeHors](https://github.com/lehors)  - Hyperledger TSC Member
+
+
```
Comment (vipinsun) [inline labs/fabric-operator.md:79]: All sponsors have to comment on this PR supporting the lab. Might be simpler to have just one or two sponsors. Five is the max I have seen(yet) in a lab. AFAIK, having a sponsor is not a requirement for a lab.

**Review by lehors — APPROVED** [review]
LGTM!

**Comment by denyeart** [comment]
LGTM, I'm happy to be a sponsor of this important lab!

**Review by mrshah-at-ibm — APPROVED** [review]

**Pull request merged**